lib/graphql/transaction.ts

```diff
@@ -120,4 +140,28 @@ export class Transaction extends IterableModel<TransactionModel> {
     const result = await this.client.rawQuery(CATEGORIZE_TRANSACTION, args);
     return result.categorizeTransaction;
   }
+
+  private parseSearchQuery(searchQuery: string): TransactionFilter {
+    const searchTerms = searchQuery.slice(0, MAX_SEARCH_QUERY_LENGTH).split(" ");
+
+    const filter: TransactionFilter = {
+      iban_likeAny: searchTerms,
+      name_likeAny: searchTerms,
+      operator: BaseOperator.Or,
+      purpose_likeAny: searchTerms,
+    };
+
+    const amountRegex = /^-?\d+([,.]\d{1,2})?$/;
+    const amountTerms = searchTerms
+      .filter((term) => amountRegex.test(term))
+      .reduce((terms: number[], term: string): number[] => {
+        const amountInCents = parseFloat(term.replace(",", ".")) * 100;
```

Comment:
what will happen if i provide `33.91`? :P

Reply:
oh shit nice catch 🤦‍♂ what if I Math.round afterwards? it should work for all cases, I'm trying to come up with a counter example

Reply:
I think `Math.round` should work in all cases: 84d001fa0a21b1e74b87c5e99108e8a05d855c7a